mod/theme_sandbox/classes/Elgg/ThemeSandbox/Menus/AdminHeader.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Elgg\ThemeSandbox\Menus;
+
+/**
+ * Event callbacks for menus
+ *
+ * @since 5.1
+ *
+ * @internal
+ */
+class AdminHeader {
+
+	/**
+	 * Register item to menu
+	 *
+	 * @param \Elgg\Event $event 'register', 'menu:admin_header'
+	 *
+	 * @return void|\Elgg\Menu\MenuItems
```

Comment:
why not ``null|\Elgg\Menu\MenuItems`` that way you can also type hint the function return value